Submodels/MTSOL_call.py

```
@@ -1,20 +1,606 @@
+
+        # Check the console output to ensure that commands are completed
+        while True:
+            # Read the output line by line
+            line = self.process.stdout.readline()
+            #print(line)
+            # Once iteration is complete, return the completed exit flag
+            if line.startswith(' =') and type == 1:
+                return ExitFlag.COMPLETED.value
+            
+            # Once the iteration is converged, return the converged exit flag
+            elif 'Converged' in line and type == 1:
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)